spec/requests/api/v1/listings/index_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe 'GET /api/v1/listings', type: :request do
+  describe 'successfully get listings' do
+    let!(:listing) { 3.times {create(:listing, category: 'car park', description: 'Wide space with safe area', scene: 'indoor', height: 4, address: 'Gothenburg', price: 200)}}
+ 
+    before do
+      get '/api/v1/listings'
+    end
+
+    it 'should return a 200 response' do
+    expect(response).to have_http_status 200
+    end
+
+    it 'should return listings' do
+    expect(response_json["listings"].count).to eq 3
+    end
+  end 
+
```

Comment:
Can we have a test to check the name of one listing and the adress of another? For example